dipy/viz/horizon/visualizer/slice.py

```diff
@@ -172,19 +174,29 @@ def _replace_picked_voxel_actor(self, x, y, z):
         self._picked_voxel_actor = actor.dot(pnt, colors=(0.9, 0.4, 0.0), dot_size=10)
         self._scene.add(self._picked_voxel_actor)
 
+    def _adaptive_percentile(self, vol_data, percentile, idx):
+        value_range = np.percentile(vol_data, percentile)
+
+        if np.sum(np.diff(value_range)) == 0:
+            warnings.warn(
+                f"The selected intensity range have no contrast for Volume N°{idx}."
+                "The selection intensities will be ignored.",
+                stacklevel=2,
+            )
+            value_range = np.percentile(vol_data, [0, 100])
```

Comment:
Also minor issue. I think this is equal to value_range = (np.min(vol_data), np.max(vol_data)). From what I've tested, using min and max is much faster. I understand from a code interpretability standpoint this is better though.